templates/resources/ec2_volume.md.tmpl

```diff
@@ -0,0 +1,27 @@
+---
+page_title: "{{.Name}} {{.Type}} - {{.ProviderName}}"
+subcategory: ""
+description: |-
+{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# {{.Name}} ({{.Type}})
+
+{{ .Description | trimspace }}
+
+## Example Usage
+
+### usage
```

Comment:
this subsection is too generic, try to add short sentence that explain the example

Reply:
Updated the changes. Now Kept only example usage and mentioned about the resources.